src/components/app-cta/app-cta.scss

```diff
@@ -4,8 +4,13 @@
   background: linear-gradient(-160deg, #f99a2a 0%, #f9902a 100%);
   padding: 5rem 0;
 
+  a {
+    line-height: 2.5rem;
+  }
+
   h3 {
     font-size: 2rem;
+    margin-left: 100px;
```

Comment:
I think this is not needed cause will push the content to the right adding extra space on the left